<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitCheck</class>
 <widget class="QMainWindow" name="FitCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_6">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Transistor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="LstTrt">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Cycle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="LstCy">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>105</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Vds</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="LstVds">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Vgs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="LstVgs">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_18">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QGroupBox" name="GrpVgs">
                <property name="title">
                 <string>YY Vs. Vgs</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="RadFitErr_4">
                   <property name="text">
                    <string>GMPoly</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="RadNoB">
                   <property name="text">
                    <string>NoB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="RadNoA">
                   <property name="text">
                    <string>NoA</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBox_3">
                   <property name="text">
                    <string>Irms</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="checkBox_4">
                   <property name="text">
                    <string>Vrms</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="RadFitErr_2">
                   <property name="text">
                    <string>FitErrB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="RadFitErr">
                   <property name="text">
                    <string>FitErrA</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QCheckBox" name="RadFitErr_3">
                   <property name="text">
                    <string>IdsPoly</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QCheckBox" name="RadFitErr_10">
                   <property name="text">
                    <string>IrmsIds</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QPushButton" name="PltVsVgs">
                   <property name="text">
                    <string>Plot</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="ChkVd0">
                   <property name="text">
                    <string>Vgs - Vd0</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="ChkIsOK">
                   <property name="text">
                    <string>Plot NOK</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="GrpVsVgsColor">
                <property name="title">
                 <string>Color On</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="2" column="2">
                  <widget class="QRadioButton" name="RadNoA_9">
                   <property name="text">
                    <string>Pass</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QRadioButton" name="RadFitErr_14">
                   <property name="text">
                    <string>Date</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="RadNoA_7">
                   <property name="text">
                    <string>Length</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QRadioButton" name="RadNoB_5">
                   <property name="text">
                    <string>W/L</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="RadFitErr_13">
                   <property name="text">
                    <string>Width</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QRadioButton" name="RadNoA_8">
                   <property name="text">
                    <string>Trt</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QRadioButton" name="RadFitErr_15">
                   <property name="text">
                    <string>Contact</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LblInfo">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_10">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="GrpFreq">
                <property name="title">
                 <string>YY Vs. Freq</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkBox_8">
                   <property name="text">
                    <string>GmMag</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBox_10">
                   <property name="text">
                    <string>PSD</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="checkBox_9">
                   <property name="text">
                    <string>GmPh</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="RadFitErr_9">
                   <property name="text">
                    <string>GMPoly</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ChkPltVsFreq">
                <property name="text">
                 <string>Update plot</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_11">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_16">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QFrame" name="frame_15">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="lineWidth">
                 <number>0</number>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QFrame" name="frame_12">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Vds</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="CmbVds"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_13">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_7">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Vgs</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QCheckBox" name="ChkVd0YY">
                      <property name="text">
                       <string>Vgs - Vd0</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="2">
                     <widget class="QDoubleSpinBox" name="SpinVgs">
                      <property name="decimals">
                       <number>3</number>
                      </property>
                      <property name="minimum">
                       <double>-1.000000000000000</double>
                      </property>
                      <property name="maximum">
                       <double>1.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.050000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QGroupBox" name="GrpYY">
                <property name="title">
                 <string>YY</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="RadFitErr_8">
                   <property name="text">
                    <string>IdsPoly</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="RadNoA_2">
                   <property name="text">
                    <string>NoA</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="RadNoB_2">
                   <property name="text">
                    <string>NoB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QCheckBox" name="RadFitErr_6">
                   <property name="text">
                    <string>FitErrB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QCheckBox" name="RadFitErr_5">
                   <property name="text">
                    <string>GMPoly</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <widget class="QCheckBox" name="RadNoA_4">
                   <property name="text">
                    <string>Ud0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="5">
                  <widget class="QCheckBox" name="RadNoB_4">
                   <property name="text">
                    <string>GmMax</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="checkBox_6">
                   <property name="text">
                    <string>Vrms</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBox_5">
                   <property name="text">
                    <string>Irms</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="4">
                  <widget class="QCheckBox" name="RadFitErr_7">
                   <property name="text">
                    <string>FitErrA</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_17">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QGroupBox" name="GrpXVar">
                <property name="title">
                 <string>XX</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="RadNoA_5">
                   <property name="text">
                    <string>Length</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QRadioButton" name="RadNoA_3">
                   <property name="text">
                    <string>Area</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QRadioButton" name="RadNoB_3">
                   <property name="text">
                    <string>W/L</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="RadFitErr_11">
                   <property name="text">
                    <string>Width</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QRadioButton" name="RadNoA_6">
                   <property name="text">
                    <string>Pass</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QRadioButton" name="RadFitErr_12">
                   <property name="text">
                    <string>Date</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButPltVsX">
                <property name="text">
                 <string>Plot</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
